src/types/UiaTextRangeBase.hpp

```diff
@@ -169,13 +115,11 @@ namespace Microsoft::Console::Types
         ~UiaTextRangeBase() = default;
 
         const IdType GetId() const noexcept;
-        const Endpoint GetStart() const noexcept;
-        const Endpoint GetEnd() const noexcept;
+        const COORD GetEndpoint(TextPatternRangeEndpoint endpoint = TextPatternRangeEndpoint::TextPatternRangeEndpoint_Start) const noexcept;

```

Comment:
It doesn't seem helpful to give this guy a default value- is that necessary? It may be clearer to always specify start or end

Reply:
Funny enough, I didn't have a single instance of using that default param haha.